README.md

```diff
@@ -2,24 +2,16 @@
 Configuration and documentation for Python Discord's Kubernetes setup!
 
 
-# How to create secrets
-Many of these deployments require you to create secret environments. These are not in this repository, for security reasons, so you need to manually create them.
+## Secrets
+We use [git-crypt](https://www.agwa.name/projects/git-crypt/) ([Github](https://github.com/AGWA/git-crypt))to secure secrets. Using this means we can commit secrets to change control, without the secrets being leaked.
 
-The easiest way to do this is to use `kubectl create secret`. For example:
+The [.gitattributes](.gitattributes) is used to determine which files to encrypt. See [git-crypt](https://www.agwa.name/projects/git-crypt/) documentation for more information.
 
-```shell script
-kubectl create secret generic new-secret-env \
-  --from-literal=MY_FAVORITE_THING=lemons \
-  --from-literal=SUPER_LAME_DOG=snoopy
-```
+To work with our secrets, you must have your GPG key added by a member of the devops team. Once that is done, you have use git-crypt as documented.
```

Comment:
Check the last sentence here.

Reply:
s/have/can https://github.com/python-discord/kubernetes/compare/8143937f27b6b131dd278d51b797473d6e92a7d0..e6f3cd69d6852c9beb148f49bb99b23dc27c0427